renderer/components/nettests/im/Telegram.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Flex,
+  Box,
+  Text
+} from 'ooni-components'
+import { FormattedMessage } from 'react-intl'
+
+import StatusBox from '../../measurement/StatusBox'
+
+const Telegram = ({measurement, isAnomaly, render}) => {
+  const testKeys = JSON.parse(measurement.test_keys)
+  let appStatus = 'Okay'
+  let webStatus = 'Okay'
+
+  if (testKeys['telegram_http_blocking'] === true) {
+    appStatus = 'Failed'
```

Comment:
So, I think the app does not work if either `telegram_http_blocking` or `telegram_tcp_blocking`. Does it make sense to you? Or, am I misreading the spec https://github.com/ooni/spec/blob/master/nettests/ts-020-telegram.md?

Reply:
You are right. I should be looking at `telegram_tcp_blocking` as well.